### ISSUE TITLE
[maui-neon]: refreshing package repo failed

### ISSUE THREAD

**Issue by star-buck** [maui-linux/issues#19]
![screenshot_20160724_210038](https://cloud.githubusercontent.com/assets/840070/17085863/3e9decce-51e3-11e6-88df-516ef68f2963.png)

**Comment by llelectronics** [comment]
It does not fail it just complains about our repo server using an unsecure keyring.
I can do nothing about it.

**Comment by star-buck** [comment]
What would make it stop complaining?

**Comment by llelectronics** [comment]
I edited the gpg.conf file on the repo server and configured our gpg key to not use sha1 anymore and prefer sha256 for signing repositories. 
I resigned the xenial repository (the other ones won't show that error message as apt on older versions doesn't complain about sha1) 

So no new key is needed. Our gpg key is good enough to also sign sha256. 
From my side everything works now without a warning. 
If it works for you too we can close this.
